### DISCUSSION TITLE
How to align content on different divs at the same vertical hight

### DISCUSSION THREAD

**Discussion by PhilipZi** [neuefische/questions#119] [Web] [answered]
Hallo zusammen, 

ich habe folgendes Problem, ich würde diese Zeiten (wie im Bild) gerne in einer Reihe darstellen, so dass diese nicht verschoben sind. 
Gibt es da eine Möglichkeit ohne, dass ich allen drei eEementen in meinem Display:Flex StyledCard eine eigene StyledComponent vergeben muss? 
Ich würde, das gerne lösen ohne zusätzliche styledcomponents zu erstellen, weiss aber leider nicht wie und ob das möglich ist. 
 
![Screenshot 2022-08-29 at 13 36 04](https://user-images.githubusercontent.com/102238180/187193088-3cc698a8-9ff0-4ce2-9f9b-95e62aa5a672.png)

hier der link zu meinem Git branch: and the Styled Component which effects the card styling and is responsible for the current style. 

https://github.com/PhilipZi/capstone-project/blob/myBranch3/src/components/StyledCards.js

**Comment by JessicaLoers** [comment, marked as answer]
**Hi @PhilipZi** 
You want the times to be vertically aligned, right 🧐

If I see it right, `<StyledTimerCards />` is the `flexbox container`  with three `flexbox items` which are aligned with `justify-content: space between`. The `flexbox  items` are evenly placed along the main axis.  Because the width of your items varies the `times` aren't  displayed vertically aligned. 

In my opinion, you won't be able to avoid "addressing" the items directly.
But I would suggest to use a grid - for example like this untested code: 

**The grid container** 
```
div {
  display: grid;
  grid-template-columns: repeat(3, 1fr);
  place-items: center;
    }
```
**Grid item `svg`** 
```
svg {
    justify-self: start;
}
```

**Grid item  `span`**  - this is your current `<p> element`
```
span {
    justify-self: end;
}
```

**Reply by PhilipZi** [reply]
Thank you!
